src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs

```diff
@@ -10,6 +10,10 @@ public abstract class CommonToastOptions
         /// </summary>
         public string ToastClass { get; set; } = Defaults.Classes.Toast;
 
+        /// <summary>
+        /// This variable applies to the title and message. If false HTML will be implemented, else HTML markup will be shown. />.
```

Comment:
Do you think we can clarify a little more the behaviour of this toggle option? Maybe using something like this:
```suggestion
        /// This variable applies to the title and message. If false the rendered HTML will be shown, else the escaped HTML markup will be shown. 
```